tests/neg/rewrite-messages.check

```diff
@@ -0,0 +1,11 @@
+-- Error: tests/neg/rewrite-messages.scala:3:18 ------------------------------------------------------------------------
+3 |import scala.util._  // error
+  |                  ^
+  |                  `_` is no longer supported for a wildcard import; use `*` instead
+  |                  This construct can be rewritten automatically under -rewrite -source future-migration.
+-- Error: tests/neg/rewrite-messages.scala:7:4 -------------------------------------------------------------------------
+7 |  2 foo 4  // error
+  |    ^^^
+  |    Alphanumeric method foo is not declared `infix`; it should not be used as infix operator.
+  |    Instead, use method syntax .foo(...) or rewrite as `foo`.
```

Comment:
Im really not sure what "or rewrite as `foo`" is supposed to mean, the original message is not clear either, but it seems that the "latter" that is rewritten automatically should be to method syntax, rather than  "rewrite as `foo`".

Reply:
I agree that backticks are the hardest problem in modern computer science. The reason we don't hear this opinion expressed on the forums is that it is inexpressible because it requires backticks to say it.

Reply:
I can see where that may be confusing. I pushed a change to try to improve the situation.